docs/devel/coding-conventions.md

```diff
@@ -33,60 +33,123 @@ Documentation for other releases can be found at
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 Code conventions
   - Bash
+
     - https://google-styleguide.googlecode.com/svn/trunk/shell.xml
-    - Ensure that build, release, test, and cluster-management scripts run on OS X
+
+    - Ensure that build, release, test, and cluster-management scripts run on
+OS X
```

Comment:
this is a weird linewrap, but it renders correctly - can we equally indent it and still render?